wordpress-mobile/WordPress-iOS
# Reader Detail: Add async featured image loading

Project Issue: #14767

### To test:
1. Launch the app
2. Tap on the Reader
3. Locate a post with a featured image
4. Enable the Network Link Conditioner to reduce the internet speed
5. Tap on the post
6. You should see the ghost loading view
7. You should see the content load in with a gray image where the featured image will be
8. After a few seconds the featured image will load in

### Notes
- I originally tries to use the ghostable view for the loading, but since those use layers they have implicit animations when the user scrolls and the view height changes the animation is very slow and does not look great. 

### PR submission checklist:
